src/en/02_devenv.md

```diff
@@ -196,7 +196,7 @@ single_line_if_else_max_width = 40
 ```
 
 For more information about the guidelines that `rustfmt` will check, have a look
-at the [Rust Style Guide](https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md).
+at the [Rust Style Guide](https://github.com/rust-lang/rust/tree/master/src/doc/style-guide/src).
```

Comment:
This used to point to an `.md`, maybe it should still:

```suggestion
at the [Rust Style Guide](https://github.com/rust-lang/rust/tree/master/src/doc/style-guide/src/README.md).
```

I wonder if we shouldn't also be versioning these links to get actual permalinks:

```suggestion
at the [Rust Style Guide](https://github.com/rust-lang/rust/tree/1.79.0/src/doc/style-guide/src/README.md).
```

Or:

```suggestion
at the [Rust Style Guide](https://github.com/rust-lang/rust/tree/master/src/doc/style-guide/src/README.md) ([permalink](https://github.com/rust-lang/rust/tree/1.79.0/src/doc/style-guide/src/README.md)).
```